app/views/username/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: dashboard_path)
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Create a username</h1>
+
+    <%= form_with model: current_user, method: :post, url: username_path do |f| %>
+      <%= f.govuk_text_field :username, label: { text: "Username" }, width: "two-thirds" %>
+      <%= f.govuk_submit "Continue" %>
+    <% end %>
+  </div>
+</div>
```

Comment:
Is there any way to reuse the existing username template for this?

Reply:
Ehh, I don't think it's worth the trouble. There are a few differences - text on submit button, header, the method that the form uses...

And the file is not very long either.

If we had more fields I would cut them out into their own partial, buuut for one field I am a bit too lazy.